### PR TITLE
Unite all tags under common structure (IDFGH-9399)

### PR DESCRIPTION
**Pull request by AndriiFilippov**
Made the logging tags lower case.

### CODE REVIEW

File: components/mdns/examples/main/mdns_example_main.c [hunk 1]
```diff
@@ -276,16 +276,15 @@ void app_main(void)
      * This is typically performed in "GOT_IP" event handler, but we call it here directly
      * since the `EXAMPLE_INTERFACE` netif is connected already, to keep the example simple.
      */
-    ESP_ERROR_CHECK(mdns_netif_action(EXAMPLE_INTERFACE, MDNS_EVENT_ENABLE_IP4));
-    ESP_ERROR_CHECK(mdns_netif_action(EXAMPLE_INTERFACE, MDNS_EVENT_ANNOUNCE_IP4));
-#endif
+    ESP_ERROR_CHECK(mdns_netif_action(EXAMPLE_INTERFACE, MDNS_EVENT_ENABLE_IP4 | MDNS_EVENT_ENABLE_IP6));
```
Comment (david-cermak) [inline components/mdns/examples/main/mdns_example_main.c:279]: Unrelated change. Probably an issue with rebasing?

**Review by david-cermak — CHANGES_REQUESTED** [review]
Please rebase

**Comment by AndriiFilippov** [comment]
> Please rebase

done

**Review by david-cermak — APPROVED** [review]

File: components/esp_websocket_client/examples/main/websocket_example.c [hunk 2]
```diff
@@ -32,7 +32,7 @@
 
 #define NO_DATA_TIMEOUT_SEC 5
 
-static const char *TAG = "WEBSOCKET";
+static const char *TAG = "websocket";
```
Comment (david-cermak) [inline components/esp_websocket_client/examples/main/websocket_example.c:35]: This need a fix in the test code, see the failure here: https://github.com/espressif/esp-protocols/actions/runs/4185017500/jobs/7256313908#step:6:292

Please update the `pytest`, as well

**Review by david-cermak — CHANGES_REQUESTED** [review]
Please fix the CI

**Review by david-cermak — APPROVED** [review]

**Pull request merged**